nautobot_golden_config/models.py

```diff
@@ -16,24 +16,12 @@
 from nautobot.extras.models.statuses import StatusField
 from nautobot.extras.utils import extras_features
 from nautobot.utilities.utils import serialize_object, serialize_object_v2
+from netutils.lib_mapper import HIERCONFIG_LIB_MAPPER_REVERSE  # pylint: disable=C0412:
```

Comment:
```suggestion
from netutils.lib_mapper import HIERCONFIG_LIB_MAPPER_REVERSE
```

Double check with `invoke pylint` on this, but I think this isn't need anymore now that it's in the standard position.